test-app/types/global.d.ts

```diff
@@ -0,0 +1,3 @@
+import 'ember-source/types';
+import 'ember-source/types/stable';
+import '@glint/environment-ember-loose';
```

Comment:
It should pull in the template registry from `ember-bootstrap`, shouldn't it?

Reply:
It's #2093 which sets up the registry, so assuming we will merge this first, we have to wait a bit. Or cherry-pick my changes? But honestly, I wouldn't overcomplicate this, we will need many more PRs to get everything properly sorted out...

Reply:
It should, but I think that since Simon implemented it in #2093 already, it should be save to omit here.
An empty registry is possible, but would just look a bit odd.